modules/overlays/test/overlays/__snapshots__/html-tooltip-overlay.test.js.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Should these be marked in gitattributes so they don't show up in diffs?

Reply:
Done